rtloader/test/rtloader/rtloader.go

```diff
@@ -31,6 +32,9 @@ func setUp() error {
 		return fmt.Errorf("make failed")
 	}
 
+	// Initialize memory tracking
+	helpers.InitMemoryTracker()
```

Comment:
Though I agree it's repeated code, I like how explicit it is to see that the test is getting tracked, so being just a single line, and perhaps a line we might not want on all tests, I think this might be fine as it currently is?